ddp-workspace/projects/ddp-angio/src/assets/i18n/en.json

```diff
@@ -272,7 +272,8 @@
                 "Title": "Your session will time-out soon",
                 "Text": "To protect your privacy, the system is going to log you out because you have been away for a while. Click “Continue” to stay logged in. Click “Sign Out” if you are done for now.",
                 "SignOut": "Sign Out",
-                "Continue": "Continue"
+                "Continue": "Continue",
+                "RenewalFailed": "Session renewal failed. Please log out and log back in again."
```

Comment:
@charlottedye going forward, how should we get things translated for Prion?

Reply:
Last I heard, we are still waiting on the final text because of the GDPR stuff.  But if your question is how we get new messages translated for Prion once Prion is in production, that is an excellent question.